spk/sickbeard-custom/src/conf/resource

```diff
@@ -0,0 +1,5 @@
+{
+	"port-config":{
+    	"protocol-file": "etc/sickbeard-custom.sc"
```

Comment:
Agree completely.  I'm just more doing proof of concepts to test packages in 6.x.  It seems to be working fairly well.

I can try and test the service tool command for the time being - however I'm also working on a new package I'd like to require 6.0 for to keep things simple....